src/QmlControls/QGCCheckBox.qml

```diff
@@ -9,6 +9,21 @@ CheckBox {
     property var __qgcPal: QGCPalette { colorGroupEnabled: enabled }
 
     style: CheckBoxStyle {
+        indicator: Rectangle {
+            implicitWidth: 16
+            implicitHeight: 16
+            radius: 3
+            border.color: control.activeFocus ? "darkblue" : "gray"
+            border.width: 1
+            Rectangle {
+                visible: control.checked
+                color: "#555"
+                border.color: "#333"
+                radius: 1
+                anchors.margins: 4
+                anchors.fill: parent
+            }
+        }
```

Comment:
Is this copied from the internal Qml style code? Wondering what happens across all OS. Since I think this style code may differ OS to OS. Not quite sure, but maybe.